package-lock.json

```diff
@@ -49712,10 +49762,12 @@
 			}
 		},
 		"react-native-video": {
-			"version": "https://raw.githubusercontent.com/wordpress-mobile/react-native-video/5.0.2-wp-2/react-native-video-5.0.2-wp-2.tgz",
-			"integrity": "sha512-JcZKEFDbje8MNEd1sEO5eQDrF7/BUaYMskI2G7rTXd0EwOPGx9rHOdx0uAmmew+NvkMNPmh+H6uKsgiaS891dA==",
+			"version": "git+https://github.com/wordpress-mobile/react-native-video.git#31ebacf5584219d4b3888b95673c35243fd30d2e",
+			"from": "git+https://github.com/wordpress-mobile/react-native-video.git#31ebacf5584219d4b3888b95673c35243fd30d2e",
```

Comment:
Not sure, will there be a problem that the `integrity` field is now missing? 🤔

Reply:
I believe that field will come back when I switch from a github commit to a tar.gz file  since [it sounds like](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json#integrity) when using a git commit just the commit hash itself is used to insure integrity. 

With that said, the static lint checks are failing because of the package-lock file, and I haven't been able to figure out what is going on with that yet.